arrows/klv/klv_0601.cxx

```diff
@@ -1001,6 +1001,23 @@ operator<<( std::ostream& os, klv_0601_sensor_fov_name value )
   os << strings[ std::min( value, KLV_0601_SENSOR_FOV_NAME_ENUM_END ) ];
   return os;
 }
+// ----------------------------------------------------------------------------
+std::ostream&
+operator<<( std::ostream& os, klv_0601_positioning_method_source value)
+{
+  static std::string strings[ KLV_0601_POSITIONING_METHOD_SOURCE_ENUM_END + 1 ] = {
+    "On-board INS",
+    "GPS",
+    "Galileo",
+    "QZSS",
+    "NAVIC",
+    "GLONASS",
+    "BeiDou-1",
+    "BeiDou-2" };
```

Comment:
You'll need one more here at the end: `"Unknown Positioning Method Source Bit"`. The`min` on the next line clamps any numbers outside the valid range to `ENUM_END`, so those will all get that `"Unknown"` string.